task/buildah/0.1/buildah.yaml

```diff
@@ -207,6 +213,13 @@ spec:
       [ -n "$COMMIT_SHA" ] && LABELS+=("--label" "vcs-ref=$COMMIT_SHA")
       [ -n "$IMAGE_EXPIRES_AFTER" ] && LABELS+=("--label" "quay.expires-after=$IMAGE_EXPIRES_AFTER")
 
+      ENTITLEMENT_PATH="/entitlement"
+      if [ -n "${ENTITLEMENT_SECRET}" ] && [ -d "$ENTITLEMENT_PATH" ]; then
```

Comment:
I guess `[ -n "${ENTITLEMENT_SECRET}" ]` is there so that you can disable the entitlement handling even if an `etc-pki-entitlement` secret exists in your namespace? (By setting the param to an empty string).

Does that not cause issues for
```yaml
  - name: etc-pki-entitlement
    secret:
      secretName: ""  # when $(params.ENTITLEMENT_SECRET) is an empty string
      optional: true
```

Reply:
You are correct, in the case of "" it is a problem. I removed the first check, as I guess it's not really needed. either the entitlement secret is in the namespace or it isn't, so checking that should be enough.